### DISCUSSION TITLE
Panel for record (project)

### DISCUSSION THREAD

**Discussion by sinnbeck** [filamentphp/filament#14154] [Help]
### Package

Panel builder

### Package Version

v3.*

### How can we help you?

Has anyone found a way to have a panel work for a certain model. I am about to make a system to manage projects. So when the user logs in, they first choose the project they want to work on and then should be presented with different pages to manage that project. I would prefer that the project id is in the url, and not session, to allow working on multiple projects at once.

I was thinking I could somehow use `$panel->path()` to set it up, but gave up for now :)

**Comment by leandrocfe** [comment]
What about using a middleware to handle this? 🤔

**Reply by sinnbeck** [reply]
That can help setting the project yes, but wouldn't add the id to the url sadly

**Comment by e2d2-dev** [comment]
hi,

you could build a custom page which includes the menu.
- change the default route to the url of this page.
- add a column to the model like is_active/is_live.
- add a redirect to the mount() function which points to the project with the active live tag
- add an action to remove the live tag to "enable" the menu again

**Reply by sinnbeck** [reply]
@e2d2-dev Thanks for the idea, but sadly I dont see how this would allow me to use a complete panel to manage a project? I plan to have several pages for managing all parts of the project, (kinda like a tenant).

**Comment by e2d2-dev** [comment]
you could also use the tenancy feature. i don't know how the rest is structured, maybe it's just a design issue

**Reply by sinnbeck** [reply]
I had concidered that, but I want it to be part of the url, so a user can work on multiple projects at once :)
